src/uct/api/uct_def.h

```diff
@@ -282,41 +281,13 @@ typedef void (*uct_unpack_callback_t)(void *arg, const void *data, size_t length
  *                               @ref uct_ep_create_sockaddr function on the
  *                               client side.
  * @param [in]  length           Length of the received data.
- * @param [out] reply_priv_data  Points to the user's data which was written in
- *                               this callback. It will be passed to reply_data
- *                               on client side upon invoking the @ref
- *                               uct_sockaddr_conn_reply_callback_t callback.
  *
- * @return  Size of the data that was written. Negative in case of an error.
+ * @return  Error code.
```

Comment:
shall we say that returning an error code will cause the sender side to call the error handler callback with error status REJECTED?

Reply:
IMO this sounds more rdmacm orionted. maybe say that an error code will cause the server side to reject to connection request from the client?

Reply:
the question is how a user of UCT will see this reject. AFAIU it will call error handling on client side, right?

Reply:
yes, so i'll mention both: server will reject, client will invoke the error handling flow.